### DISCUSSION TITLE
NURBS

### DISCUSSION THREAD

**Discussion by luiswirth** [NicolasKlenert/enterpolation#8] [Q&A] [answered]
Is the `BSpline` type actually a NURBS?

**Comment by NicolasKlenert** [comment, marked as answer]
Yes, `BSpline` supports NURBS (Non Uniform Rational B-Spline). The non uniform part are just knots which are not equdistant from each other and the rational part can be achieved via weights. There is a simple [example](https://github.com/NicolasKlenert/enterpolation/blob/main/examples/nurbs.rs) in the repositrory.

I will add a remark in the bspline module that they are able to support NURBS.

**Reply by luiswirth** [reply]
Okay thanks, that clears things up.
Yeah that note, would be helpful :).
